guides/v2.2/release-notes/release-notes-2-2-10-commerce.md

```diff
@@ -0,0 +1,794 @@
+---
+group: release-notes
+title: Magento Commerce 2.2.10 Release Notes
+
+---
+
+*Patch code and release notes published on October 8, 2019.*
+
+Magento Commerce 2.2.10 offers significant platform upgrades, substantial security changes, and PSD2-compliant core payment methods. This release includes over 147 functional fixes and enhancements to the core product and 75 security enhancements. It includes 56 community-fixed GitHub issues.
+
+## Highlights
+
+Look for the following highlights in this release:
+
+### Platform upgrades
+
+The following upgrades to core platform components boost platform security and support PCI compliance:
+
+* Magento 2.2.10 now supports PHP 7.2.x (tested with 7.2.21). <!--- MC-16174 -->
+* Magento 2.2.10 does not support PHP 7.0.x.  <!--- MC-18521 -->
+  
+### Substantial security enhancements
+
+This release includes the following security enhancements:
+
+* PSD2 compliance for core payment methods  
+* Fixes for 75 critical security issues
+* Significant platform-security enhancements that boost XSS (cross-site scripting) protection against future exploits. This effort is the culmination of several months of concentrated effort on Magento's part to reduce our backlog of security enhancements.
+
+#### Core payment methods integrations are now compliant with PSD2 regulations
+
+The European Union recently revised the Payment Services Directive (PSD) regulation with an updated version–PSD2. This revised regulation will go into effect on September 14, 2019 or shortly thereafter, and will significantly affect  most payment processing involving credit cards or bank transfers.  See the Magento Forum DevBlog post [3D Secure 2.0 changes](https://community.magento.com/t5/Magento-DevBlog/3D-Secure-2-0-changes/ba-p/136460) for more information on Magento Payment Provider Recommendations and a wealth of links to PSD2 regulation discussions.
+
```

Comment:
General note -- Check for double-spacing between words.

Reply:
Did a global search and replace in both files